src/handlers/Handler.ts

```diff
@@ -9,10 +9,10 @@ abstract class Handler<
   protected _inputType: Input;
   protected _outputType: Output;
   /**
-   * This is the timeout used for the handler.
-   * If it is not set then the default timeout time for the `RPCServer` is used.
+   * This is the handlerTimeout used for the handler.
+   * If it is not set then the default handlerTimeout time for the `RPCServer` is used.
    */
-  public timeout?: number;
+  public handlerTimeout?: number;
```

Comment:
Why rename this? I think it's fine to call it the `timeout` property of the handler.

Reply:
timeout gets confusing in a larger code base with so many timeouts, specific name is better